posthog/api/activity_log.py

```diff
@@ -77,8 +77,7 @@ class ActivityLogViewSet(TeamAndOrgViewSetMixin, viewsets.GenericViewSet, mixins
     pagination_class = ActivityLogPagination
 
     def filter_queryset_by_parents_lookups(self, queryset) -> QuerySet:
-        team = self.team
-        return queryset.filter(Q(organization_id=team.organization_id) | Q(team_id=team.id))
+        return queryset.filter(team_id=self.team.pk)
```

Comment:
should it be `self.team.id` instead? or is the `pk` an alias for `id`? what if change the `pk` at some point? :D

Reply:
Ah, yeah, it's stylistic

* pk always point to the tables primary key whatever it is
* normally that is id

I genuinely don't have a strong opinion - but I'm sure other people used to 🤔

Reply:
> To sum up, it is up to you to choose whether to use field name id or the pk shortcut. If you are not developing a library for Django and use [automatic primary key fields](https://docs.djangoproject.com/en/2.1/topics/db/models/#automatic-primary-key-fields) for all models, it is safe to use id everywhere, which is sometimes faster.

random SO discussion https://stackoverflow.com/a/53100893

so, I guess pk protects you against the id not being called id in the future and this'd have to change anyway - whereas id means you don't have to think about that unless you rename the id field 🙃

Reply:
switched it...

I feel like I've absorbed someone else's preference and can't justify it... === `habit into the bin`